django/core/exceptions.py

```diff
@@ -71,6 +71,11 @@ class ImproperlyConfigured(Exception):
     pass
 
 
+class FilePermissionError(Exception):
```

Comment:
If we need a new exception, it should be documented in https://docs.djangoproject.com/en/dev/ref/exceptions/